### DISCUSSION TITLE
find tutorial for socket io

### DISCUSSION THREAD

**Discussion by Ahmad-s-s** [miguelgrinberg/python-socketio#854] [General]
Hello 
I'm looking for some tutorial for my project.
the subject of my project is tutorial, does any body know where can i find some ?

**Comment by miguelgrinberg** [comment]
Search on Google and YouTube. There's tons of Socket.IO tutorials.

**Comment by atomscaler** [comment]
The python-socketio documentation is very comprehensive and has many
examples which are more than enough to get started with and understand the
basics:
https://python-socketio.readthedocs.io/en/latest/index.html


On Tue, 18 Jan 2022 at 13:35, Ahmad-s-s ***@***.***> wrote:

> Hello
> I'm looking for some tutorial for my project.
> the subject of my project is tutorial, does any body know where can i find
> some ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/miguelgrinberg/python-socketio/discussions/854>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHVT3Q2A42SJILC745RA63LUWVUBJANCNFSM5MHGBPVQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
____________________

Ben Dyer

t: 07788 871 719
e: ***@***.***
